lib/Site.js

```diff
@@ -255,6 +255,7 @@ Site.prototype.createPage = function (config) {
     src: config.pageSrc,
     title: config.title || '',
     titlePrefix: this.siteConfig.titlePrefix,
+    depthOfHeadings: this.siteConfig.depthOfHeadings || 3,
```

Comment:
Let's not use magic numbers. Specify a constant at the top of the file. Maybe `DEPTH_OF_HEADINGS_DEFAULT`?